commands/mr/update/mr_update.go

```diff
@@ -49,6 +50,19 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 				}
 			}
 
+			if cmd.Flags().Changed("reviewer") {
+				givenReviewers, err := cmd.Flags().GetStringSlice("reviewer")
+				if err != nil {
+					return err
+				}
+				ur = cmdutils.ParseAssignees(givenReviewers)
+
+				err = ur.VerifyAssignees()
```

Comment:
It's good you used these existing function instead of creating new functions since they would perform almost the same task. We have to change the function names from `ParseAssignees` and `VerifyAssignees` to `ParseAssigneesOrReviewers` and `VerifyAssigneesOrReviewers` repectively. You can choose any other function name that best describes the purpose of the functions.

Reply:
GitLab web calls it assignment so let's maintain the function names